lib/u3d/installation.rb

```diff
@@ -176,6 +213,10 @@ def version
       version
     end
 
+    def build_number
+      @build_number ||= LinuxInstallationHelper.new.find_build_number(root_path)
```

Comment:
Maybe we could make the `find_build_number` a class method rather than an instance? It would make a bit more sense to me. Especially given the fact that the Helper doesn't carry any sort of data so all of its current methods could be moved to self?

Reply:
I was trying to avoid class methods! I don't find them particularly elegant from an API point of view. Thoses classes are meant internal though, not part of the public API in my mind.